File: 4_A_VBnet/Interval.vb
Public Class Interval
    Public LowerInclusiveBound As Integer
    Public IntervalStep As Integer
    Public Count As Integer = 0
    Public RelativeFrequency As Double
    Public Percentage As Double

    Public Function containsValue(ByVal v As Double) As Boolean
        Return (v >= LowerInclusiveBound AndAlso v < (LowerInclusiveBound + IntervalStep))
    End Function

End Class

File: 15_A/Interval.cs
public partial class Interval
{
    public double LowerInclusiveBound;
    public double Step;
    public int Count = 0;

    public double RelativeFrequency;
    public double Percentage;

    public bool containsValue(double v) => (v >= LowerInclusiveBound && v < (LowerInclusiveBound + Step));

    public string printInterval()
    {
        return $"[{LowerInclusiveBound} - {LowerInclusiveBound + Step})";
    }
}

